pyproject.toml

```
@@ -19,8 +19,13 @@ pyproj = "^3.6.1"
 numpy = "^2.0.0"
 metpy = "^1.6.2"
 prometheus-client = "^0.20.0"
-
 mysql-connector = "^2.2.9"
+tabulate = "^0.9.0"
+ncepbufr = [
+	{ platform = "linux_x86_64", file = "./third_party/NCEPLIBS-bufr/wheel_dist/ncepbufr-12.0.1-py312-none-linux_x86_64.whl" },
+	{ platform = "macosx_14_0_arm64", file = "./third_party/NCEPLIBS-bufr/wheel_dist/ncepbufr-12.0.1-py312-none-macosx_14_0_arm64.whl" }
+]
```

It looks like Poetry's `platform` field takes 1 of `darwin`, `linux`, or `win32`. To get at the architecture Poetry supports [PEP 508 `markers`](https://peps.python.org/pep-0508/#environment-markers). You'd do: `marker = "platform_machine == 'arm64'"` for ARM, for example.

I also bumped the version number to match the new nceplibs version in the recommendation below.

```suggestion
ncepbufr = [
	{ platform = "linux", markers = "platform_machine == 'x86_64'", file = "./third_party/NCEPLIBS-bufr/wheel_dist/ncepbufr-12.1.0-py312-none-linux_x86_64.whl" },
	{ platform = "darwin", markers = "platform_machine == 'arm64'", file = "./third_party/NCEPLIBS-bufr/wheel_dist/ncepbufr-12.1.0-py312-none-macosx_14_0_arm64.whl" }
]
```

---

thanks for catching this.